lib/easy_meli/errors.rb

```diff
@@ -9,10 +9,12 @@ def initialize(message, response)
   end
 
   class AuthenticationError < Error; end
-  
+
+  class InvalidTokenError < Error; end
```

Comment:
How about if we create errors for each variant? I think it would be useful for tracking in general and we can make AccessTokenError and AuthenticationError parents.

```ruby

  class AuthenticationError < Error; end
  class InvalidGrantError < AuthenticationError; end
  class ForbiddenError < AuthenticationError; end

  class AccessTokenError < Error; end
  class InvalidTokenError < AccessTokenError; end
  
```
In the code that parses the response you could do something like
```ruby
ERROR_MAP = { 'invalid grant' => [InvalidGrantError, 'Invalid Grant'], ... }
```